runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewDelegate.java

```diff
@@ -35,9 +38,14 @@
     private static boolean sLibraryLoaded = false;
     private static boolean sRunningOnIA = true;
     private static final String PRIVATE_DATA_DIRECTORY_SUFFIX = "xwalkcore";
+    // In previous versions, MANDATORY_PAKS is actually not mandatory, it's just a list of
```

Comment:
Dead code usually rots very quickly. How about just mentioning what happened in the commit message?